### ISSUE TITLE
Reversing video clip does not work when exported

### ISSUE THREAD

**Issue by koztudott** [OpenShot/openshot-qt#702]
i7-4790 @ 3.6 GHz, 16 GB RAM
Windows 8.1 Pro
OpenShot Version: 2.3.3

- Imported video into new project
- Placed it as a clip
- Set Time > Normal > Backwards > 1X
- Verified in the preview window that it works (plays backwards)
- Exported the project
- In the resulting video file (mp4, h.264, same profile as the original video), the clip is not reversed, but plays forward

**Comment by koztudott** [comment]
Probably a duplicate of #667

**Comment by Daveland3D** [comment]
Same issue. Reversed video does not render in reverse when exported to mp4 format.

Windows 8.1
OpenShot Version: 2.3.4

Imported video into new project
Placed it as a clip
Set Time > Normal > Backwards > 1X
Verified in the preview window that it works (plays backwards)
Exported the project
In the resulting video file (mp4, h.264, same profile as the original video), the clip is not reversed, but plays forward

**Comment by DredderGun** [comment]
Absolutely same problem! I did same steps

Ubuntu 16.04
OpenShot Version: 2.3.4

**Comment by DylanC** [comment]
@koztudott  - Did you try this in the v2.4.1 release?

**Comment by koztudott** [comment]
I've tried it and it's working well in 2.4.1 !

**Comment by Daveland3D** [comment]
Did you try it after exporting?

David Landrecht, PLS

Professional Land Surveyor

Daveland 3D Services, LLC

258 Sumac Street

Philadelphia, PA

19128-3826

**Comment by koztudott** [comment]
Yes, I've repeated the same steps as before, and with this new version the exported video was also reversed.

**Comment by Daveland3D** [comment]
Awesome!
Thanks!

David Landrecht, PLS

Professional Land Surveyor

Daveland 3D Services, LLC

258 Sumac Street

Philadelphia, PA

19128-3826
